tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs

```diff
@@ -98,7 +98,7 @@ public void WhenRuntimeParametersArePassedToInstallScripts(string runtime, strin
         [InlineData("2.2", "dotnet")]
         [InlineData("3.0", "dotnet")]
         [InlineData("3.1", "dotnet")]
-        // [InlineData("5.0", "dotnet")] - Broken
+        [InlineData("5.0", "dotnet")]
```

Comment:
Shouldn't `"release/5.0", "dotnet"` work as well, now that 5.0 GA'ed?

Reply:
no, it's still broken